java/impl/gcp/src/main/java/co/worklytics/psoxy/CloudFunctionRequest.java

```diff
@@ -51,7 +53,11 @@ public String getHttpMethod() {
     @Override
     @SneakyThrows
     public byte[] getBody() {
-        return request.getInputStream() == null ? request.getInputStream().readAllBytes() : null;
+        if (body == null) {
+            body = request.getInputStream() != null ? request.getInputStream().readAllBytes() : null;
```

Comment:
Coming from https://github.com/Worklytics/psoxy/pull/151; to properly handle a body in request/response. Mainly for Dropbox.